src/components/ButtonGroup/ButtonGroup.module.css

```diff
@@ -3,31 +3,37 @@
 \*------------------------------------*/
 
 /**
- * 1) A group of buttons displayed in a horizontal fashion
+ * A group of buttons displayed in an organized fashion.
  */
 .button-group {
-  display: flex;
-  align-items: center;
-  gap: var(--eds-size-2);
+  display: inline-grid;
+  grid-auto-flow: column;
 }
 
 /**
- * Responsive button group
- * 1) A button group that's stacked by default then moves side-by-side
- *    when space becomes available 
+ * Spacing variants.
  */
-.button-group--responsive {
-  @media all and (max-width: $eds-bp-md) {
-    align-items: flex-start;
-    flex-direction: column;
-  }
+.button-group--spacing-1x {
+  gap: var(--eds-size-1);
+}
+.button-group--spacing-max {
+  width: 100%;
+  justify-content: space-between;
+}
+
+/**
+ * Vertical orientation variant.
+ */
+.button-group--vertical {
+  grid-auto-flow: row;
 }
 
 /**
- * Stacked button group
+ * Vertical spacing max variant.
+ * Hopefully no one actually ever uses this... 
  */
-.button-group--stacked {
-  display: flex;
-  flex-direction: column;
-  align-items: flex-start;
+.button-group--spacing-max.button-group--vertical {
+  width: 100%;
+  height: 100%;
+  align-content: center;
```

Comment:
Is this to prevent breaking styling? Should we even include if we don't want people using it this way?

Reply:
Yeah, I was just trying to cover our bases. I can't see a use case where this is needed, but I didn't like the idea that there was a combo you could make with the props that we just didn't do even though it's not difficult to implement. But I say it's up to you – if you feel strongly that it should go, remove it and update the warning to say it's not supported.

Reply:
I can kind of see a case where on a vertical bar, there might be buttons spaced max and placed vertically.... (not a designer lol). Will keep